src/components/styles/EventTileStyles.js

```diff
@@ -4,13 +4,13 @@ import colors from '../../styles/colors';
 const styles = StyleSheet.create({
   eventName: {
     textAlign: 'center',
-    fontSize: 30,
-    color: colors.titlegrey,
-    fontWeight: '500',
+    fontSize: 20,
+    color: 'black',
```

Comment:
small change and insignificant, but in the future use colors only from our constants colors file